src/librustdoc/templates/page.html

```diff
@@ -7,20 +7,20 @@
     <meta name="description" content="{{page.description}}"> {#- -#}
     <meta name="keywords" content="{{page.keywords}}"> {#- -#}
     <title>{{page.title}}</title> {#- -#}
-    <link rel="preload" as="font" type="font/woff2" crossorigin href="{{static_root_path | safe}}SourceSerif4-Regular.ttf.woff2"> {#- -#}
-    <link rel="preload" as="font" type="font/woff2" crossorigin href="{{static_root_path | safe}}FiraSans-Regular.woff2"> {#- -#}
-    <link rel="preload" as="font" type="font/woff2" crossorigin href="{{static_root_path | safe}}FiraSans-Medium.woff2"> {#- -#}
-    <link rel="preload" as="font" type="font/woff2" crossorigin href="{{static_root_path | safe}}SourceCodePro-Regular.ttf.woff2"> {#- -#}
-    <link rel="preload" as="font" type="font/woff2" crossorigin href="{{static_root_path | safe}}SourceSerif4-Bold.ttf.woff2"> {#- -#}
-    <link rel="preload" as="font" type="font/woff2" crossorigin href="{{static_root_path | safe}}SourceCodePro-Semibold.ttf.woff2"> {#- -#}
+    <link rel="preload" as="font" type="font/woff2" crossorigin href="{{static_root_path|safe}}SourceSerif4-Regular.ttf.woff2"> {#- -#}
```

Comment:
Does Askama not accept spaces around the `|` or is this a style preference?

Reply:
The Askama parser, for better or worse, parses `|` with spaces around it as the `BitOr` operator, so these changes are necessary. That does seem a little finicky and I don't think I was very intentional about it when I wrote that code, but IIRC it's fairly common style in Jinja templates to forgo spacing around the filter operator.

FWIW, I like to have spaces inside the braces; I'd find this more readable as `{{ static_root_path|safe }}`, but I didn't want to include stylistic changes in (the first version of) this PR.